File: infosupport/infoserver/src/main/java/app/models/GP.java
package app.models;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "General_practitioner")
public class GP {

  @Id
  @Column(name = "big_code")
  private String big_code;

  @Column(name = "specialty")
  private String specialty;

  @Column(name = "user_id")
  private Integer user_id;

  public GP() {

  }

  public String getBig_code() {
    return big_code;
  }

  public void setBig_code(String big_code) {
    this.big_code = big_code;
  }

  public String getSpecialty() {
    return specialty;
  }

  public void setSpecialty(String specialty) {
    this.specialty = specialty;
  }

  public void setUser_id(Integer user_id) {
    this.user_id = user_id;
  }
}

File: infosupport/src/app/models/gp.ts
import {User} from "./user";

export class GP {

  private _big_code: number;
  private _specialty: string;


  constructor() {
  }

  get big_code(): number {
    return this._big_code;
  }

  set big_code(value: number) {
    this._big_code = value;
  }

  get specialty(): string {
    return this._specialty;
  }

  set specialty(value: string) {
    this._specialty = value;
  }
}
